test/test_tree_2d_advection.jl

```diff
@@ -83,7 +88,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
   @trixi_testset "elixir_advection_timeintegration.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_timeintegration.jl"),
       l2   = [2.4976030518356626e-5],
-      linf = [0.0005531580316338533])
+      linf = [0.0005531580316338533],
+      coverage_override = (maxiters=10^5, tspan=(0.0, 0.1)))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
To cover `methods_2N.jl`. I need to let the simulation terminate by time instead of `maxiters` to cover a specific part of the code.